### PR TITLE
fix: table evaluation metrics fix calculations when no tables found in predictions

### PR DESCRIPTION
**Pull request by pawel-kmiecik**
The current way table structure metrics are computed does not cover cases when none table is found and all stats are empty.

This PR fixes this + adds some hardenning tests for table eval processor.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 * **Fix Google Drive source key** Allow passing string for source connector key.
 * **Fix table structure evaluations calculations** Replaced special value `-1.0` with `np.nan` and corrected rows filtering of files metrics basing on that.
 * **Fix Sharepoint-with-permissions test** Ignore permissions metadata, update test.
+* **Fix table structure evaluations for edge case** Fixes the issue when the prediction does not contain any table - no longer errors in such case.
```
Comment (badGarnet) [inline CHANGELOG.md:19]: don't forget to bump version number

**Review by badGarnet — APPROVED** [review]
